inventory_management_system_api/models/units.py

```diff
@@ -5,14 +5,30 @@
 from pydantic import BaseModel, ConfigDict, Field
 
 from inventory_management_system_api.models.custom_object_id_data_types import StringObjectIdField
+from inventory_management_system_api.models.mixins import CreatedModifiedTimeInMixin, CreatedModifiedTimeOutMixin
 
 
-class UnitOut(BaseModel):
+class UnitBase(BaseModel):
+    """
+    Base database model for a Unit
+    """
+
+    value: str = Field(description="Value of the Unit")
+    # Used for uniqueness checks (sanitised value)
+    code: str = Field(description="Code of the Unit")
```

Comment:
```suggestion
    value: str
    # Used for uniqueness checks (sanitised value)
    code: str
```
We don't normally put descriptions in models.